### ISSUE TITLE
Fix not displayed structure level in LoS

### ISSUE THREAD

**Issue by Elblinator** [OpenSlides/openslides-autoupdate-service#866]
**Current behaviour:**
If a participant is only allowed to put themselves on a LoS (is **not** allowed to manage or see the LoS)  their own structure level is not displayed 

**Reproduction:**
1. project something onto the autopilot
2. navigate to the LoS
3. change the deligates group so that they cannot see other participants on the LoS
4. connect structure level with countdown
5. put user A (with one structure level) on the LoS 
6. as admin you can see the structure level from the participant
7. login as User A
8. Navigate to the LoS 
9. you cannot see your own structure level

**autoupdate from the delegate:**
```[autoupdate] from stream: autopilot_content_7:subscription 557252489 
(2) [{…}, {…}]0: 
agenda_item: 
{17: {…}}
list_of_speakers: 
{18: {…}}
meeting: 
{2: {…}}
meeting_user: 
{6: {…}}
projection: 
{7: {…}}
projector: 
{3: {…}}
speaker: 
{22: {…}}
topic: 
{10: {…}}
user: 
{2: {…}}
```

**autoupdate from the admin:**
```
[autoupdate] from stream: autopilot_content_7:subscription 386092060 
(2) [{…}, {…}]
0: 
agenda_item: 
{17: {…}}
list_of_speakers: 
{18: {…}}
meeting: 
{2: {…}}
meeting_user: 
{5: {…}, 6: {…}, 7: {…}}
projection: 
{7: {…}}
projector: 
{3: {…}}
speaker: 
{18: {…}, 21: {…}, 22: {…}}
structure_level: 
{4: {…}, 5: {…}, 6: {…}}
structure_level_list_of_speakers: 
{1: {…}, 2: {…}, 3: {…}}
topic: 
{10: {…}}
user: 
{1: {…}, 2: {…}, 3: {…}}
```

**Wanted behaviour:**
As a participant you should be able to see your own structure level 
Note: You should only be able to see the structure level **countdown** IF you are allowed to manage or see the LoS

**Comment by jsangmeister** [comment]
Seems to be that the definition for the countdown (meaning `structure_level_list_of_speakers`) is already correct, that's why it was only a note by @Elblinator - feel free to correct me though ;) For the structure level, I would just add a suffix to the "can see" definition: "... OR the user has a meeting user which has the requested structure level set", or would that impact the performance too much? Because that is basically what @Elblinator wrote above.

**Comment by jsangmeister** [comment]
You are right, if we want to keep the old structure level behaviour from when it was a simple string, this would be the logical conclusion. So the new rule would be:
```
The user can see a structure level if he has `list_of_speakers.can_see` OR he can see any of the meeting users from `meeting_user_ids`.
```
> What about the `structure_level_list_of_speakers`. Currently, this depends on the permission `list_of_speakers.can_see`. Maybe it can be seen, if the user [can see the linked list_of_speaker](https://github.com/OpenSlides/openslides-autoupdate-service/blob/80c2139e4729dfe7cc0d30c44ba7606e6cc16d22/internal/restrict/collection/list_of_speakers.go#L11-L16)?

And I also agree with this change ;)

**Comment by Elblinator** [comment]
> You are right, if we want to keep the old structure level behaviour from when it was a simple string, this would be the logical conclusion. So the new rule would be:
> 
> [...]
>
> And I also agree with this change ;)

I agree!

**Comment by Elblinator** [comment]
Yes 
Thank you
